instrumentation/jaxrs/jaxrs-2.0/jaxrs-2.0-testing/src/main/groovy/JaxRsFilterTest.groovy

```diff
@@ -27,17 +28,35 @@ abstract class JaxRsFilterTest extends AgentInstrumentationSpecification {
 
   abstract makeRequest(String url)
 
+  Tuple2<String, String> runRequest(String resource) {
+    if (testRunsOnServer()) {
+      return makeRequest(resource)
+    }
+    // start a trace because the test doesn't go through any servlet or other instrumentation.
+    return runUnderServerTrace("test.span") {
+      makeRequest(resource)
+    }
+  }
+
+  boolean testAbortPrematch() {
+    true
+  }
+
+  boolean testRunsOnServer() {
```

Comment:
the current name `testX` makes it looks like the others, which are optional things to test

```suggestion
  boolean runsOnServer() {
```

Reply:
done